python-for-beginners/08 - Handling conditions/README.md

```diff
@@ -14,8 +14,8 @@ else:
 [Comparison operators](https://docs.python.org/3/library/stdtypes.html#comparisons)
 
 - < less than
-- < greater than
+- \> greater than             # \(quote grammar) is added problem related to markdown grammar escape
```

Comment:
Please give me feedback about added comment if it is appropriate or not.
This comment was added for users who might wonder why \ is added

Reply:
That's not markdown commenting grammar.

Reply:
What should i correct them into?
Not sure about what "\" does, so i added comment

Reply:
It's a markdown file, so if you want to comment, follow the markdown grammar.
Maybe most of the people won't look at the raw markdown text.
If you still want to leave comments, google `markdown comment` and fix it to suitable grammar.

Reply:
If you want to look what this REAMDME.md looks like, just press `view file` button at the `files changed` menu. [Direct link](https://github.com/20-1-SKKU-OSS/c9-python-getting-started/blob/33bf281b4a2cd615605e50b6ac85037244b39b12/python-for-beginners/08%20-%20Handling%20conditions/README.md)

Reply:
Hmm.. I'll just delete it!